Opcodes/sf.h

```diff
@@ -46,9 +46,9 @@ struct _splitType {
         int64_t endOffset;
         int64_t startLoopOffset;
         int64_t endLoopOffset;
-        char overridingRootKey;
-        char coarseTune;
-        char fineTune;
+        /*char*/ SBYTE overridingRootKey;
```

Comment:
I think it might be better to remove the commented /*char*/ throughout this commit and just have the SBYTE.

Reply:
ok

Reply:
ok